nixos/lib/testing.nix

```diff
@@ -188,6 +188,7 @@ in rec {
             networking.hostName = "client";
             nix.readOnlyStore = false;
             virtualisation.writableStore = false;
+            system.stateVersion = "19.09"; # TODO: determine automatically
```

Comment:
I think this TODO ought to be resolved before merging this, as otherwise we'll forget it after 19.09 branch-off, and we'll start testing outdated versions of postgresql on the next postgresql bump (or similar thing that would involve bumping stateVersion)

Reply:
Giving it a bit more thought: either that or just add bumping this to the release process checklist, maybe -- definitely need to record the existence of this TODO somewhere at least, though.